qemu/qemu.go

```diff
@@ -1474,6 +1485,21 @@ func (config *Config) appendKnobs() {
 			config.qemuParams = append(config.qemuParams, "-device")
 			config.qemuParams = append(config.qemuParams, deviceMemParam)
 		}
+	} else if config.Knobs.FileBackedMem == true {
```

Comment:
is it mutual exclusive, with the previous `if...else...` statements (MemPrealloc and HugePages ) If not I would remove the `else`

Reply:
It's mostly the `dimmName` part that makes them exclusive, for the same reason `HugePages` and `FileBackedMem` are exclusive to each other.